sdk/resourceconnector/Azure.ResourceManager.ResourceConnector/tests/Azure.ResourceManager.ResourceConnectorManagementTestBase.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.ResourceConnector.Tests
+{
+    public class Azure.ResourceManager.ResourceConnectorManagementTestBase : ManagementRecordedTestBase<Azure.ResourceManager.ResourceConnectorManagementTestEnvironment>
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md to add tests for all resources.

Reply:
Added tests - working on getting tag updated on `assests.json`

Reply:
Pushed the `assests.json` tag